references/segmentation/presets.py

```diff
@@ -1,39 +1,40 @@
 import torch
-import transforms as T
+from torchvision.prototype import features, transforms as T
 
 
-class SegmentationPresetTrain:
-    def __init__(self, *, base_size, crop_size, hflip_prob=0.5, mean=(0.485, 0.456, 0.406), std=(0.229, 0.224, 0.225)):
-        min_size = int(0.5 * base_size)
-        max_size = int(2.0 * base_size)
+class WrapIntoFeatures(T.Transform):
+    def forward(self, sample):
+        image, segmentation_mask = sample
+        return image, features.SegmentationMask(segmentation_mask.squeeze(0), dtype=torch.int64)
```

Comment:
Why aren't we wrapping Image? This is what we currently do on the Classification references.

Reply:
The `ToImageTensor` already wraps into a `features.Image`:

https://github.com/pytorch/vision/blob/24890d718f5a73586ef093371912b5b37a5b0d46/torchvision/prototype/transforms/functional/_type_conversion.py#L30

Thus, `segmentation_mask` here is already a `features.Image` and we just re-wrap in the correct type. Maybe `WrapIntoFeatures` is a bad name. Let's resolve this together with https://github.com/pytorch/vision/pull/6433#discussion_r970876270.

Reply:
Sounds good. Ping me when you are OK to check and start using it. Thanks for the work!